apps/experiments/views/experiment.py

```diff
@@ -594,21 +595,36 @@ def form_valid(self, form):
         is_default = form.cleaned_data["is_default_version"]
         working_version = Experiment.objects.get(id=self.kwargs["experiment_id"])
 
-        if working_version.is_archived:
-            raise PermissionDenied("Unable to version an archived experiment.")
```

Comment:
what this check intentionally removed?

Reply:
No. Must have been a bad merge. Thanks for spotting this